platform/android/detect.py

```diff
@@ -119,7 +119,9 @@ def mySpawn(sh, escape, cmd, args, env):
 			env.extra_suffix=".armv7.neon"+env.extra_suffix
 		else:
 			env.extra_suffix=".armv7"+env.extra_suffix
-
+        elif env['android_arch']=='armv8':
```

Comment:
In the bottom right corner of the Kate window there is "Tab size: 8". Click on it to configure tabs for indentation and no spaces, so that it does not insert spaces automatically against your will.